### ISSUE TITLE
Cannot select subscription other than LTT

### ISSUE THREAD

**Issue by jdhodge** [bmlzootown/Hydravion#9]
I'm happy to have a floatplane channel on Roku (Thank you for your hard work in providing this!), but I am having a problem (on 1.5). On my Roku Ultra, I can open the app fine, and if I select the LTT subscription things work fine, but if I select the BitWit Ultra channel, it makes a bong sound and shows the checkmark to the left, but then nothing happens.  I can arrow down and pick LTT, it makes the same sound, the checkmark moves, then it opens the LTT sub, but then if I go back, arrow back up to the BitWit Ultra sub and select it, it makes the sound and moves the checkmark, but again nothing else. Please let me know if there's anything I can provide for troubleshooting purposes.

**Comment by bmlzootown** [comment]
I've added in some checks to make sure the livestream info is actually there, which could have been a potential issue, and have switched from using a channel's title (used to check whether they're live or not) to the urlname provided by the API, which should prevent any encoding issues with whitespaces/special characters.

Tested it on my end, and things seem to be working now, so I pushed out an update. Should be able to update normally through Roku's system update in settings. (That's a lot of "updates" in just two sentences. Yikes!)

Let me know if that resolves the issue for you. Thanks!
